modules/ort/src/main/kotlin/workflow/analyzers/OrtResultAnalyzer.kt

```diff
@@ -37,6 +40,11 @@ class OrtResultAnalyzer : ManualAnalyzer() {
 
     override fun getName() = "OrtResult"
 
+    override fun configure(configMap: MutableMap<String, String>?) {
```

Comment:
The signature should ideally be `configMap: Map<String, String>` if `configMap` is never written to and is guaranteed to never be `null`.